module/documents/item.mjs

```diff
@@ -90,10 +90,17 @@ export default class Item5e extends SystemDocumentMixin(Item) {
      */
     if ( options.pack || options.parent?.pack ) Hooks.callAll("dnd5e.initializeItemSource", this, data, options);
 
-    ActivitiesTemplate.initializeActivities(data);
```

Comment:
I noticed there are some times when `_initializeSource` might receive partial source data, such as when creating a temporary Document, i.e. `new Actor({ name: 'foo', type: 'npc' });`. I moved this to `fromSource` instead, which generally expects to instantiate a Document from full source data. It's not foolproof, but it's the best I can think of right now.

Reply:
Seems to be working, but I'll keep an eye out to see if anything funky happens.